src/helpers/markdown-helpers.js

```diff
@@ -117,29 +172,59 @@ const getChapterTitle = chapter => {
   return fs.readFileSync(titleFile, 'utf8').replace(/^\s+|\s+$/g, '');
 };
 
+const createRelativePath = (fromFile, toFile) => {
+  const relativePath = path.relative(fromFile, toFile);
+  //needs to remove the first relative sign as we are dealing with files and not folder
+  if (relativePath.startsWith('..')) {
+    return relativePath.substr('../'.length);
+  }
+  return relativePath;
+};
 const generateSummaryFileTitle = summaryFileName => {
   writeToFile(summaryFileName, config.SUMMARY_FILE_TITLE);
 };
 
-const generateSummaryFile = (content, summaryFileName, parentDirectory = '', currentLevel = 0) => {
+const generateSummaryFile = (
+  content,
+  summaryFileName,
+  createFileLink = true,
+  parentDirectory = '',
+  currentLevel = 0
+) => {
   debug(`Generating summary file: ${summaryFileName}`);
-
+  const tab = '  ';
   const level = getNextLevel(currentLevel);
+
   content.forEach(item => {
     if (fileShouldBeIgnored(item)) {
       return;
     }
-    if (item.type === 'directory') {
-      const link = path.join(parentDirectory, item.name, `${item.name}.md`);
-      const tab = '  ';
-      writeToFile(summaryFileName, `${tab.repeat(level)}* [${getChapterTitle(item)}](${link})`);
-      if (hasChildren(item)) {
-        generateSummaryFile(item.children, summaryFileName, path.join(parentDirectory, item.name), level);
-      }
+    if (item.type !== 'directory') {
+      return;
+    }
+
+    const title = getChapterTitle(item);
+    const link = createFileLink ? path.join(parentDirectory, item.name, `${item.name}.md`) : `#${createAnchor(title)}`;
+
+    writeToFile(summaryFileName, `${tab.repeat(level)}* [${title}](${link})`);
+
+    if (hasChildren(item)) {
+      generateSummaryFile(item.children, summaryFileName, createFileLink, path.join(parentDirectory, item.name), level);
     }
   });
 };
 
+// Returns a valid markdown anchor from a header that can be used to navigate within a single document
+// code taken from https://gist.github.com/asabaylus/3071099#gistcomment-2563127 for completion
+const createAnchor = val =>
```

Comment:
@Yuri05 This was missing. In short, the anchor are generated automatically using some weird conventiosn (e.g. some char are removed, everything is lower case.. etc). I found a method that seems to do it all.